test/ui-test/testSuites/suite_settings/tst_userIdentity/test.feature

```diff
@@ -2,10 +2,11 @@ Feature: User Identity
 
     As a user I want to set my identity, that is: display name, bio and social links.
 
-    Background: Sign up and land
-        Given A first time user lands on the status desktop and generates new key
-        When user signs up with username "tester123" and password "TesTEr16843/!@00"
-        Then the user lands on the signed in app
+
+    The feature start sequence is the following (setup on its own `bdd_hooks`):
+    ** given A first time user lands on the status desktop and generates new key
+    ** when user signs up with username "tester123" and password "TesTEr16843/!@00"
+    ** and the user lands on the signed in app
 
 	@mayfail
```

Comment:
Is this mayfail still accurate? Can you add a comment to explain why.

Reply:
Oh, yess!! It is still accurated and the reason is that the scenario is broken due to the new user profile design. It will be addressed in task #8281. Also added comment in code.